page-list.php

```
@@ -170,7 +171,9 @@ function siblings_unqprfx_shortcode( $atts ) {
 		$list_pages = wp_list_pages( $page_list_args );
 
 		$return .= $pagelist_unq_settings['powered_by'];
-		if ($list_pages) {
+		if (!( !empty( $offset ) && is_numeric( $offset ) && $offset_count > $offset & $post->post_parent) 
```

PHPMD (2.15.0)

Avoid unused local variables such as '$offset_count'. (Unused Code Rules)

(UnusedLocalVariable)